Simplify and fix example for the reflection article (#1602)
dotnet/docs-desktop: dotnet-desktop-guide/framework/wpf/graphics-multimedia/how-to-create-a-reflection.md

---
# How to: Create a Reflection

> This example shows how to use a <xref:System.Windows.Media.VisualBrush> to create a reflection. Because a <xref:System.Windows.Media.VisualBrush> can display an existing visual, you can use this capability to produce interesting visual effects, such as reflections and magnification.

Reviewer: ```suggestion
This article shows how to use a <xref:System.Windows.Media.VisualBrush> to create a reflection. Because a <xref:System.Windows.Media.VisualBrush> can display an existing visual, you can use this capability to produce interesting visual effects, such as reflections and magnification.
```

:::image type="content" source="media/how-to-create-a-reflection/reflection.png" alt-text="A XAML element with latin text and two circles, whose upside down reflection is displayed below it.":::

## XAML example

The following example uses a <xref:System.Windows.Media.VisualBrush> to create a reflection of a <xref:System.Windows.Controls.Border> that contains several elements.

:::code language="xaml" source="./snippets/how-to-create-a-reflection/xaml/MainWindow.xaml" id="Reflection":::

## Code-based example

> The following example uses a <xref:System.Windows.Media.VisualBrush> to create a reflection of a <xref:System.Windows.Controls.Border> that contains several elements, when setting the content of a `Window` during the `Window.Loaded` event:

Author: ```suggestion
The following example uses a <xref:System.Windows.Media.VisualBrush> to create a reflection of a <xref:System.Windows.Controls.Border> that contains several elements. This code is run in the `Window.Loaded` event handler:
```
